timeline-chart/src/layer/time-graph-chart.ts

```diff
@@ -618,8 +625,26 @@ export class TimeGraphChart extends TimeGraphChartLayer {
             if (el instanceof TimeGraphStateComponent && !this.mousePanning && !this.mouseZooming) {
                 this.selectState(el.model);
             }
-            if (this.mouseInteractions && this.mouseInteractions.click) {
-                this.mouseInteractions.click(el, e);
+            
+            var clickCount = 1;
+            var self = this;
+            this._mouseClicks++;
+
+            if (this._mouseClicks === 1) {
+                this._recentlyClickedGlobal = cloneDeep(e.data.global);
+                this._dblClickTimer = setTimeout(function(){
```

Comment:
I would suggest to use debounce(). This way, you can keep clicking fast and it won't go back to 1 in the middle of the multi-click.

Note, the debounce should be called for all clicks not just the first one. So the only thing special to do on first click is to store the global position.